Gemfile.lock

```diff
@@ -413,7 +413,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  buildkite-test_collector (~> 2.1)
+  buildkite-test_collector (~> 2.2)
```

Comment:
I updated the `Gemfile` in #465 to `2.2` but forgot to commit the `bundle install` changes.